### ISSUE TITLE
Molly doing a full circle before starting trajectory

### ISSUE THREAD

**Issue by antoinealb** [cvra/molly-the-motion-planner#13]
Sorry, could not find a descriptive title.

Basically I am playing a bit with Molly to generate a trajectory to test Tracy. What I tried is simply generating a straight trajectory with no obstacles, to observe the ramps.

This is my test script:
```python
from molly.Pathplanner import get_path
from molly.Settings import Settings
from molly.Vec2D import Vec2D
import matplotlib.pyplot as plt

settings = Settings(static_poly_obs=[])

start_pos = Vec2D(1., 1.)
dest_pos = Vec2D(2., 1.)
start_heading = Vec2D(1., 0.)
start_v = 0.

path = get_path(settings, [], [], start_pos, start_heading,
                start_v, dest_pos)

# Workaround issue 12
path = filter(lambda p: len(p) == 4, path)

x = [p[0].pos_x for p in path]
y = [p[0].pos_y for p in path]

plt.plot(x, y)
plt.plot([start_pos.pos_x, dest_pos.pos_x], [start_pos.pos_y, dest_pos.pos_y], 'rx')
plt.xlabel("x (m)")
plt.ylabel("y (m)")
plt.legend(["molly output", "start/end points"])

plt.axis([0, 3, 0, 2])
plt.show()
```

![figure_1](https://cloud.githubusercontent.com/assets/2419961/7338519/81c1f85c-ec4e-11e4-8db6-3116aad5de82.png)

**Comment by antoinealb** [comment]
The problem doesn't happen when the robot is not facing the starting point: 
![molly](https://cloud.githubusercontent.com/assets/2419961/7338562/b23423c4-ec4f-11e4-9441-3394d8961821.png)

**Comment by 31415us** [comment]
should be fixed in #17. please verify once it's been merged (=
